src/unsorted/faq.md

```diff
@@ -0,0 +1,167 @@
+# Frequently Asked Questions
+
+- [Does Rust support my device][#does-rust-support-my-device]
+
+## Does Rust support my device?
+
+### Short answer
+
+As of 2018-09-18 the Rust compiler supports cross compiling to these embedded
+architectures (see `rustup target list`):
+
+- ARM Cortex-M (since 1.27)
+  - `thumbv6m-none-eabi`, Cortex-M0
+  - `thumbv7m-none-eabi`, Cortex-M3
+  - `thumbv7em-none-eabi`, Cortex-M4 and Cortex-M7
+  - `thumbv7em-none-eabihf`, Cortex-M4F and Cortex-M7F
+- ARM Cortex-R (1.30-beta)
+  - `armebv7r-none-eabi`, big endian Cortex-R4 and Cortex-R5
+  - `armebv7r-none-eabihf`, big endian Cortex-R4F and Cortex-R5F
+  - `armv7r-none-eabi`, little endian Cortex-R4 and Cortex-R5
+  - `armv7r-none-eabihf`, little endian Cortex-R4F and Cortex-R5F
+- ARM Linux
+  - ARMv5TE, ARMv6, ARMv7 and ARMv8 in GNU and MUSL flavors and in soft float
```

Comment:
Can we give some examples here, like we did above with the Cortex-M<foo>? Like the ARM11 is an ARMv6 (relevant, as it's on a Raspberry Pi Zero/1), while an ARM926EJ-S is ARMv5, Cortex-A8 is ARMv7-A (sic) and Cortex-A53 is ARMv8-A (sic).

Reply:
Also worth noting that that ARMv4 (ARM7) is not supported. I mean, the most recent model came out 17 years ago, but some may be interested in e.g. Dreamcast homebrew.

Reply:
Done